src/components/field-types/FileFieldComponent.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <validate :state="state" :custom="{'validate': isValid}">
+    <div class="form-group">
+      <label :for="field.id">{{ field.label }}</label>
+    </div>
+
+    <div class="form-group">
+      <div class="custom-file">
+        <label class="custom-file-label" :for="field.id">
+          {{ label }}
+        </label>
+
+        <input
+          :id="field.id"
+          :name="field.id"
+          :required="isRequired"
+          class="custom-file-input"
```

Comment:
add form-control-lg ?

Reply:
due to the custom-file-input class, it does not behave like a standard form-control.
So adding the lg class will have no effect

Reply:
But now the field is smaller (less high) than the other fields, this looks inconsistent.

Reply:
implementing https://github.com/molgenis/molgenis-ui-form/issues/70 would fix this